Content.Shared/DoAfter/DoAfterComponent.cs

```diff
@@ -0,0 +1,105 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.DoAfter;
+
+[RegisterComponent, NetworkedComponent]
+public sealed class DoAfterComponent : Component
+{
+    public readonly Dictionary<byte, DoAfter> DoAfters = new();
```

Comment:
These should be datafields at some point and DoAfter a datadefinition to support pause / saving correctly.